.github/workflows/docs.yml

```diff
@@ -17,15 +17,17 @@ jobs:
       - name: Setup mdBook
         uses: peaceiris/actions-mdbook@v1
         with:
-          mdbook-version: 'latest'
+          mdbook-version: "latest"
+      - name: Install callouts preprocessor
+        run: cargo install --git https://github.com/ToolmanP/rs-mdbook-callout
```

Comment:
Is there a way to instead install a specific crate? Also, how does this affect local build? For example, we'd have to tell the users to the documentation system how to install this...